### PR TITLE
turtle game (reach the beach)

### PR DESCRIPTION
**Pull request by Eevi0n**
# Your checklist for this pull request

## Author name 

**Daniel Dunavetsky**

## About your game

**15 levels of a baby turtle trying to reach the ocean and not getting eaten**

**press WASD to move and if you want to restart press i press l to start and reach the ocean complete the level(first level isplayed twice to understand controls) **

## Code
Check off the items that are true. 
- [x] The game was made using the [Sprig editor](https://sprig.hackclub.com/editor).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [x] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [x] The name of the image matches the name of your file. 

> Thanks for your PR!

### CODE REVIEW

**Comment by Eevi0n** [comment]
My friend went to maker faire and told me about it since she knew I like making games

**Review by LucasHT22 — CHANGES_REQUESTED** [review]
Wow Maker Faire! Did you win the game? I didn't haha

**Comment by Eevi0n** [comment]
which game? I didnt go to maker faire or am I not understanding

**Comment by Eevi0n** [comment]
sorry for the dumb question but this is my first pull request so I don't know how it works but how do i see the requested changes that were asked for?

**Comment by LucasHT22** [comment]
> which game? I didnt go to maker faire or am I not understanding

Oh sorry, I meant your sprig game :)

**Comment by Eevi0n** [comment]
O ya i beat it once but i feel like it is a bit harder then I intended :)

File: games/turtle game (reach the beach).js [hunk 1]
```diff
@@ -0,0 +1,440 @@
+
```
Comment (LucasHT22) [inline games/turtle game (reach the beach).js:1]: like this

**Comment by LucasHT22** [comment]
Please check this: The name of the file/game contains only alphanumeric characters, -s, or _s. :)

**Comment by Eevi0n** [comment]
I think I fixed it

**Comment by LucasHT22** [comment]
Could you rename the image too? Thanks:)

**Comment by Eevi0n** [comment]
sorry im having trobule changing the name of a png do i need to like delete it and then reupload it or is there a way to do it on github

**Comment by LucasHT22** [comment]
Don't apologize!! You need to delete and reupload:)

**Comment by Eevi0n** [comment]
I cant upload the new png it say
"File uploads require push access to this repository."

**Comment by LucasHT22** [comment]
Try on your fork, if you can't make it either, I can merge and then you make other PR with the updated version or you send to me and I add

**Comment by Eevi0n** [comment]
Ya I can not if I try to edit the PNG the button is grayed out

**Review by LucasHT22 — APPROVED** [review]
Probably some bug, please make other PR with the update:) Congrats:)
If you're a teen, you can get your sprig here! https://sprig-order.hackclub.dev/

Hack Clubbers would love to see this! Just post a link and 1-2 sentences talking about your game to our [#ship](https://hackclub.slack.com/archives/C0M8PUPU6) [channel](https://hackclub.com/slack/):)

**Pull request merged**